docs/docs/administration/customize.md

```diff
@@ -1154,11 +1154,13 @@ defaultWelcomeMessage=<default welcome message>
 defaultWelcomeMessageFooter=<default welcome message footer>
 ```
 
-When a front-end creates a BigBlueButton session, it may also pass a `welcome` parameter in the [create](/development/api#create) API call.
+OBS: Welcome messages are now rendered in Markdown format.
 
-The final welcome message shown to the user (as blue text in the Chat window) is a composite of `welcome` + `defaultWelcomeMessage` + `defaultWelcomeMessageFooter`.
+When a front-end creates a BigBlueButton session, it may also pass a `welcomeMessage` parameter in the [create](/development/api#create) API call.
 
-The welcome message is fixed for the duration of a meeting. If you want to see the effect of changing the `welcome` parameter, you must [end](/development/api#end) the current meeting or wait until the BigBlueButton server removes it from memory (which occurs about two minutes after the last person has left). You can overwrite these values in file `/etc/bigbluebutton/bbb-web.properties`, just remember to restart BigBlueButton with `sudo bbb-conf --restart` for the new values to take effect.
+The final welcome message shown to the user (as blue text in the Chat window) is a composite of `welcomeMessage` + `defaultWelcomeMessage` + `defaultWelcomeMessageFooter`.
```

Comment:
It's not true.
It will show `welcomeMessage` and in case it is missing then it will show `defaultWelcomeMessage`.